front-end/src/components/AppNavBar.jsx

```diff
@@ -55,6 +55,9 @@ export default class NavbarComp extends React.Component {
           <NavbarToggler onClick={this.toggle}/>
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav className="ml-auto" navbar>
+            <NavItem onClick={this.close}>
```

Comment:
Should this `NavItem` only appear when the user is logged in? If so, we can move this component to be beside the `Log Out` link

Reply:
done. `Log Out` is kinda weird now though (shows hovering over text, but the button still works)

<img width="174" alt="Screen Shot 2021-02-24 at 1 39 24 AM" src="https://user-images.githubusercontent.com/32996140/108973109-3b03f900-7641-11eb-958a-b357642806b7.png">

Reply:
Sweet! The logout part is kind of weird (that's on me, I need to figure out how to get it to work with Reactstrap), so I can fix it up at a later point